example/ndpiSimpleIntegration.c

```diff
@@ -74,6 +75,8 @@ struct nDPI_flow_info {
     struct ndpi_flow_struct * ndpi_flow;
     struct ndpi_id_struct * ndpi_src;
     struct ndpi_id_struct * ndpi_dst;
+
+    char info[160];
```

Comment:
This member isn't used anywhere except when setting he first element to '\0'.

Reply:
Yes, you are right. That was used; after it is not used any more. I will remove.